ansible_bender/db.py

```diff
@@ -158,8 +158,11 @@ def _get_and_bump_build_id(data):
         """ return id for next build id and increment the one in DB """
         next_build_id = data["next_build_id"]
         data["next_build_id"] += 1
-        # TODO: verify such build is not in DB, we don't want to overwrite, just in case
-        return str(next_build_id)
+        if not data["builds"].get[next_build_id]:
```

Comment:
you also probably mean to do `data["builds"].get(next_build_id_str)`

Reply:
Resolved 😄